src/ethereum_test_execution/get_blobs.py

```diff
@@ -0,0 +1,78 @@
+"""Test execution format to get blobs from the execution client."""
+
+from typing import ClassVar, Dict, List
+
+from ethereum_test_base_types import Hash
+from ethereum_test_forks import Fork
+from ethereum_test_rpc import BlobAndProof, EngineRPC, EthRPC
+from ethereum_test_types import NetworkWrappedTransaction, Transaction
+
+from .base import BaseExecute
+
+
+def versioned_hashes_with_blobs_and_proofs(
+    tx: NetworkWrappedTransaction,
+) -> Dict[Hash, BlobAndProof]:
+    """
+    Return a dictionary of versioned hashes with their corresponding blobs and
+    proofs.
+    """
+    versioned_hashes: Dict[Hash, BlobAndProof] = {}
+    for blob in tx.blobs:
+        versioned_hash = blob.versioned_hash()
+        if blob.kzg_proof is not None:
+            versioned_hashes[versioned_hash] = BlobAndProof(blob=blob.data, proof=blob.kzg_proof)
+        elif blob.kzg_cell_proofs is not None:
+            versioned_hashes[versioned_hash] = BlobAndProof(
+                blob=blob.data, proofs=blob.kzg_cell_proofs
```

Comment:
there can be many proofs?  easy to make typo

Reply:
Created `BlobAndProofV1` and `BlobAndProofV2` and it's indeed much cleaner, thanks!